scripts/fast-reboot

```diff
@@ -109,15 +122,15 @@ docker kill teamd > /dev/null
 # Kill swss dockers
 docker kill swss
 
+# syncd service stop is capable of handling both warm/fast/cold shutdown
+systemctl stop syncd
```

Comment:
Possible risk with these 2 lines.

`docker kill swss` will trigger swss service stop(), and will `docker stop syncd`. Then it will trigger syncd service stop().

But you manually `systemctl stop syncd` again.

Reply:
I tested with following code:

> #!/bin/bash
> 
> docker kill swss
> echo $?
> 
> sudo systemctl stop syncd
> echo $?

 sonic prompt # ./test.sh 
swss
0
0

Stopping a service after it has been stopped is no-op. Is this your concern or are you thinking about something else?